pubsub/jetstream/jetstream.go

```diff
@@ -132,8 +132,11 @@ func (js *jetstreamPubSub) Subscribe(req pubsub.SubscribeRequest, handler pubsub
 			js.l.Debugf("Processing JetStream message %s/%d", m.Subject,
 				jsm.Sequence)
 			opErr := handler(js.ctx, &pubsub.NewMessage{
-				Topic: m.Subject,
+				Topic: req.Topic,
```

Comment:
Why's this. being changed to `req.Topic` but the message field is used in the below line?
should it also be changed in the previous line?
Quick question is the message subject generally different from topic?

Reply:
nvm saw the issue description ...